main.js

```diff
@@ -0,0 +1,138 @@
+var form = document.getElementsByClassName('post-form')[0];
+var i = 0
+
+form.addEventListener('submit', function(event1) {
+  event.preventDefault()
+  var userName = $('#name').val();
+  var post = $('#message').val();
+
+  // create a new cell in the table with the name and post content and add a quality column and a editing column
+  var newDiv = document.createElement('div');
+  newDiv.id = i
+  newDiv.className = "Rtable message"
+  $('.posts')[0].append(newDiv)
+  var postName = "<div class='Rtable-cell Rtable-cell-medium Rtable-cell-content'><span class='cell-content'>" + userName + "</span></div>"
```

Comment:
Using fewer variables would make this more readable, perhaps. But it's also sort of just how things go before we start using things like Handlebars.